From 0e7b3b659a88fe901d255c78122d74f50b794461 Mon Sep 17 00:00:00 2001
Message-Id: <0e7b3b659a88fe901d255c78122d74f50b794461.1349175436.git.minovotn@redhat.com>
In-Reply-To: <94968b7fa9b14e71f004474d7ce77e189e6a2bf3.1349175436.git.minovotn@redhat.com>
References: <94968b7fa9b14e71f004474d7ce77e189e6a2bf3.1349175436.git.minovotn@redhat.com>
From: Amos Kong <akong@redhat.com>
Date: Mon, 1 Oct 2012 14:12:40 +0200
Subject: [PATCH 27/34] socket: clean up redundant assignment

RH-Author: Amos Kong <akong@redhat.com>
Message-id: <1349100767-9066-8-git-send-email-akong@redhat.com>
Patchwork-id: 42566
O-Subject: [RHEL-6.4 qemu-kvm PATCH v7 07/14] socket: clean up redundant assignment
Bugzilla: 680356
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>

(Cherry-picked from commit 8715fc1e4caee09bde28f8e6844ad1a3bfe52f0e)

Signed-off-by: Amos Kong <akong@redhat.com>
Signed-off-by: Stefan Hajnoczi <stefanha@linux.vnet.ibm.com>
---
 qemu-sockets.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qemu-sockets.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/qemu-sockets.c b/qemu-sockets.c
index c5e20ff..91554e2 100644
--- a/qemu-sockets.c
+++ b/qemu-sockets.c
@@ -281,7 +281,6 @@ int inet_connect_opts(QemuOpts *opts, Error **errp)
                         inet_strfamily(e->ai_family),
                         e->ai_canonname, uaddr, uport, strerror(errno));
             closesocket(sock);
-            sock = -1;
             continue;
         }
         freeaddrinfo(res);
-- 
1.7.11.4

